index.html

```diff
@@ -58,7 +60,221 @@ <h3>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+    <div class="work-2">
+        <div class="proj-work">
+
+       
+        <section class="projects">
+            Profesional Art Printing Data
+         </section>
+         <section class="projects-details">
+             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         </section>
+         <section class="langs-2">
+             <div>html</div>
+             <div>bootstrap</div>
+             <div>ruby</div>
+         </section>
+         <section class="proj-btn">
+             <button class="see">see project</button>
+         </section>
+      </div>
+    </div>
+  </section>
 </main>
 
+<div class="about">
+    <section class="about-details">
+        <section class="abt-container">
+
+    
+            <div class="abt">
+         
+          
+             <h1>
+                About me
+             </h1>
+          
+             <div class="abt-details">
+             <p>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+               
+             </p> 
+             <section class="abt-btn">
+                <button class="get-btn">Get My Resume</button>
+            </section>  
+             </div>
+            
+         </div>
+         <hr> 
+        </section>
+         </section>
+    <section class="about-details">
+  <div class="languages">
+            <ul>
+                <li><h4><span class="kite"></span>Language</h4></li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+        </div>
+
+    </section>
+    <section class="about-details">
+        <div class="languages">
+            <ul>
+                <li><h4><span class="sqr"></span>Frameworks</h4></li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapYBara</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+    </section>
+    <section class="about-details">
+        <div class="languages">
+            <ul>
+                <li><h4><span class="circle"></span>Skills</h4></li>
+                <li>CodeKit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+    </section>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be a ul list, each of them in a li

- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li
![image](https://user-images.githubusercontent.com/43698511/129449876-193b1228-41f9-4232-bf32-4a851437a304.png)

- [ ] As you can see in the picture the content is not centered as it should. And there's in an extra space that shouldn't be there. Kindly fix the issue.